backend/account/views.py

```diff
@@ -191,10 +191,25 @@ def user_folder(request, user_id, fid):
     except Folder.DoesNotExist:   # Wrong id
         return HttpResponse(status=401)
 
+    my_posts = Post.objects.filter(folder_id=fid)
+
     post_in_folder_ids = PostInFolder.objects.filter(folder=folder).values_list('post_id', flat=True)
     posts_in_folder = Post.objects.filter(id__in=list(post_in_folder_ids))
 
+    place_in_folder_ids = PlaceInFolder.objects.filter(folder=folder).values_list('place_id', flat=True)
+    places_in_folder = Place.objects.filter(id__in=list(place_in_folder_ids))
+
     response_dict = {
+        'my_posts': [ {
+            'id': post.id,
+            'thumbnail_image': post.thumbnail_image.url if post.thumbnail_image else None,
```

Comment:
How about assigning empty string(`''`) instead of `None` when there is no image?
I think this value would be managed in front-end as a string, so maintaining same type between frontend-backend  could reduce the confusions and unexpected errors.

Reply:
I get it! Thank you :)